cmd/influxd/launcher/launcher.go

```diff
@@ -658,14 +658,14 @@ func (m *Launcher) run(ctx context.Context) (err error) {
 	)
 
 	tenantStore := tenant.NewStore(m.kvStore)
-	ts := tenant.NewService(tenantStore)
+	ts := tenant.NewSystem(tenantStore, m.log.With(zap.String("store", "new")), m.reg, metric.WithSuffix("new"))
 
 	var (
-		userSvc         platform.UserService                = tenant.NewUserLogger(m.log.With(zap.String("store", "new")), tenant.NewUserMetrics(m.reg, ts, metric.WithSuffix("new")))
```

Comment:
If you want you could just use the value from the struct below and save the var declarations. up to you